src/lib.rs

```diff
@@ -67,8 +72,12 @@ impl AttestationService {
 
     /// Create a new Attestation Service, and connect to a remote rvps.
     #[cfg(feature = "rvps-proxy")]
-    pub async fn new_with_rvps_proxy(rvps_addr: &str) -> Result<Self> {
-        let _config = Config::default();
+    pub async fn new_with_rvps_proxy(rvps_addr: &str, config_path: Option<&str>) -> Result<Self> {
```

Comment:
A little question: Why not we pass `Config` struct here? Sometimes a user might want to change the config in  the code

Reply:
Good suggestion, I have changed it.